setup.py

```diff
@@ -181,7 +181,7 @@ def build_extensions(self):
     "entry_points": {
         "pennylane.plugins": ["lightning.qubit = pennylane_lightning:LightningQubit",],
     },
-    "description": "PennyLane is a Python quantum machine learning library by Xanadu Inc.",
+    "description": "PennyLane lightning.qubit plugin",
```

Comment:
(Minor suggestion): Just in case we'd have more devices/would have more functionality not necessarily connected to `lightning.qubit`?
```suggestion
    "description": "PennyLane-Lightning plugin",
```